package.json

```diff
@@ -2,33 +2,54 @@
   "name": "woocommerce-gutenberg-emails",
   "version": "0.1.0",
   "description": "This is a feature plugin to enable Gutenberg for editing emails.",
-  "scripts": {
-    "precommit": "node bin/pre-commit-hook.js",
-    "prepush": "node bin/pre-push-hook.js"
-  },
   "husky": {
     "hooks": {
       "pre-commit": "node bin/pre-commit-hook.js",
       "pre-push": "node bin/pre-push-hook.js"
     }
   },
+  "author": "Automattic",
+  "license": "GPL-2.0-or-later",
+  "keywords": [],
+  "homepage": "https://github.com/woocommerce/woocommerce-gutenberg-emails#readme",
   "repository": {
     "type": "git",
     "url": "git+https://github.com/woocommerce/woocommerce-gutenberg-emails.git"
   },
-  "author": "Automattic",
-  "license": "GPL-2.0-or-later",
   "bugs": {
     "url": "https://github.com/woocommerce/woocommerce-gutenberg-emails/issues"
   },
-  "homepage": "https://github.com/woocommerce/woocommerce-gutenberg-emails#readme",
+  "engines": {
+    "node": ">=8.0.0",
+    "npm": ">=6.0.0"
+  },
+  "dependencies": {
+    "core-js": "^2.6.5"
+  },
   "devDependencies": {
+    "@wordpress/blocks": "^6.2.4",
+    "@wordpress/editor": "^9.2.4",
+    "@wordpress/element": "^2.3.0",
+    "@wordpress/i18n": "^3.3.0",
+    "@wordpress/scripts": "^3.1.0",
     "chalk": "^2.4.2",
     "child_process": "^1.0.2",
-    "husky": "^1.3.1"
+    "clean-webpack-plugin": "^2.0.1",
+    "css-loader": "^2.1.1",
+    "eslint-plugin-jest": "^22.5.1",
+    "husky": "^1.3.1",
+    "mini-css-extract-plugin": "^0.6.0",
+    "style-loader": "^0.23.1"
   },
-  "engines": {
-    "node": ">=8.0.0",
-    "npm": ">=6.0.0"
+  "scripts": {
+    "build": "wp-scripts build",
+    "check-engines": "wp-scripts check-engines",
+    "check-licenses": "wp-scripts check-licenses --production",
+    "lint:css": "wp-scripts lint-style '**/*.css'",
+    "lint:js": "wp-scripts lint-js .",
+    "lint:pkg-json": "wp-scripts lint-pkg-json .",
+    "start": "wp-scripts start",
+    "test:e2e": "wp-scripts test-e2e",
+    "test:unit": "wp-scripts test-unit-js"
```

Comment:
It looks like `test:e2e` & `test:unit` are doing the same thing currently… I'm not really familiar with how the `test:e2e` script is meant to work, though.

Reply:
Right, I removed `test:e2e` for now (643d08c9a40a1c53c0dd7b3ce1387943bd145503). We can add it back in the future if we add e2e tests.